backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/StubBuilder.kt

```diff
@@ -5,21 +5,32 @@
 
 package org.jetbrains.kotlin.backend.konan.objcexport
 
-internal class StubBuilder {
-    private val children = mutableListOf<Stub<*>>()
+internal class StubBuilder<S : Stub<*>>(private val relaxedMode: Boolean) {
```

Comment:
"Relaxed mode" is kinda too vague (especially in classes that just pass this flag down).
Maybe something like "ignore exceptions" or "ignore errors"?
Introducing a enum for exception handling policy is also a good choice here.

Reply:
Reworked the PR. As discussed, I suggest we extend the existing warning collector functionality to include handling of exceptions as well.